src/index.ts

```diff
@@ -62,6 +62,31 @@ class IconFontBuildr {
 
     this.config = mergeWith ( {}, this.configDefault, config, ( prev, next ) => isArray ( next ) ? next : undefined );
 
+    this.config.icons = Object.keys ( this.config.icons ).reduce( ( obj, val ) => {
+        const iconConfig = this.config.icons [ val ],
+              key = iconConfig.icon || iconConfig
+
+        obj [ key ] = {
+          icon: key,
+          name: iconConfig.name || key,
+          // default values:
+          codepoints: undefined,
```

Comment:
perhaps the default value here should be an empty array rather than `undefined`.

Reply:
Yeah I guess that wouldn't hurt.